src/api/goods.ts

```diff
@@ -3,17 +3,35 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+function sortFiveFirst(array: Good[]) {
+  const sorted = [...array]
+    .sort((firstItem, secondItem) => (
+      firstItem.name.localeCompare(secondItem.name)
+    ));
+
+  sorted.length = 5;
```

Comment:
Consider using `slice` instead of manually setting the length. This avoids modifying the original array and is more concise. Example: `sorted.slice(0, 5);`